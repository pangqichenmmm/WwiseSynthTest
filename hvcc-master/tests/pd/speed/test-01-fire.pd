#N canvas 156 160 424 248 10;
#X obj 317 76 hip~ 1000;
#X obj 124 76 bp~ 1200 0.6;
#X obj 217 76 bp~ 2600 0.4;
#X obj 37 76 bp~ 600 0.2;
#N canvas 769 179 475 599 firegen 0;
#N canvas 8 22 441 619 crackles 0;
#X obj 62 515 *~;
#X obj 126 314 random 30;
#X obj 126 358 line~;
#X obj 126 386 *~;
#X obj 94 446 + 1500;
#X obj 62 470 bp~ 4000 1;
#X obj 93 424 * 500;
#X obj 62 540 outlet~;
#X obj 63 52 inlet~;
#X msg 126 337 1 \, 0 15;
#X obj 126 270 delay 1000;
#X obj 126 290 t b b;
#X obj 126 170 log;
#X obj 126 130 random 2000;
#X obj 126 150 / 2000;
#X obj 126 100 loadbang;
#X obj 126 250 t b f;
#X obj 126 230 min 3000;
#X obj 126 210 max 40;
#X obj 126 190 * -400;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 1 0 6 0;
#X connect 2 0 3 0;
#X connect 2 0 3 1;
#X connect 3 0 0 1;
#X connect 4 0 5 1;
#X connect 5 0 0 0;
#X connect 6 0 4 0;
#X connect 8 0 5 0;
#X connect 9 0 2 0;
#X connect 10 0 11 0;
#X connect 11 0 1 0;
#X connect 11 1 13 0;
#X connect 12 0 19 0;
#X connect 13 0 14 0;
#X connect 14 0 12 0;
#X connect 15 0 13 0;
#X connect 16 0 10 0;
#X connect 16 1 10 1;
#X connect 17 0 16 0;
#X connect 18 0 17 0;
#X connect 19 0 18 0;
#X restore 57 96 pd crackles;
#N canvas 0 22 450 300 hissing 0;
#X obj 106 188 *~;
#X obj 141 57 lop~ 1;
#X obj 142 107 *~;
#X obj 66 58 hip~ 1000;
#X obj 141 133 *~;
#X obj 142 80 *~ 10;
#X obj 141 157 *~ 600;
#X obj 105 222 outlet~;
#X obj 111 27 inlet~;
#X connect 0 0 7 0;
#X connect 1 0 5 0;
#X connect 2 0 4 0;
#X connect 2 0 4 1;
#X connect 3 0 0 0;
#X connect 4 0 6 0;
#X connect 5 0 2 0;
#X connect 5 0 2 1;
#X connect 6 0 0 1;
#X connect 8 0 1 0;
#X connect 8 0 3 0;
#X restore 149 96 pd hissing;
#N canvas 0 22 170 271 lapping 0;
#X obj 21 109 clip~ -0.9 0.9;
#X obj 21 129 hip~ 25;
#X obj 21 86 hip~ 25;
#X obj 21 40 bp~ 30 5;
#X obj 21 65 *~ 100;
#X obj 21 153 *~ 0.6;
#X obj 20 177 outlet~;
#X obj 21 12 inlet~;
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 2 0 0 0;
#X connect 3 0 4 0;
#X connect 4 0 2 0;
#X connect 5 0 6 0;
#X connect 7 0 3 0;
#X restore 246 97 pd lapping;
#X obj 148 61 noise~;
#X text 203 60 factor common noise;
#X text 305 129 mix levels;
#X obj 151 154 outlet~;
#X obj 57 123 *~ 0.2;
#X obj 246 123 *~ 0.6;
#X obj 150 122 *~ 0.3;
#X text 144 30 fire generator;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 2 0 8 0;
#X connect 3 0 0 0;
#X connect 3 0 1 0;
#X connect 3 0 2 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 6 0;
#X restore 37 46 pd firegen;
#X obj 37 146 *~ 0.2;
#X obj 37 176 dac~;
#N canvas 769 179 475 599 firegen 0;
#N canvas 8 22 441 619 crackles 0;
#X obj 62 515 *~;
#X obj 126 314 random 30;
#X obj 126 358 line~;
#X obj 126 386 *~;
#X obj 94 446 + 1500;
#X obj 62 470 bp~ 4000 1;
#X obj 93 424 * 500;
#X obj 62 540 outlet~;
#X obj 63 52 inlet~;
#X msg 126 337 1 \, 0 15;
#X obj 126 270 delay 1000;
#X obj 126 290 t b b;
#X obj 126 170 log;
#X obj 126 130 random 2000;
#X obj 126 150 / 2000;
#X obj 126 100 loadbang;
#X obj 126 250 t b f;
#X obj 126 230 min 3000;
#X obj 126 210 max 40;
#X obj 126 190 * -400;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 1 0 6 0;
#X connect 2 0 3 0;
#X connect 2 0 3 1;
#X connect 3 0 0 1;
#X connect 4 0 5 1;
#X connect 5 0 0 0;
#X connect 6 0 4 0;
#X connect 8 0 5 0;
#X connect 9 0 2 0;
#X connect 10 0 11 0;
#X connect 11 0 1 0;
#X connect 11 1 13 0;
#X connect 12 0 19 0;
#X connect 13 0 14 0;
#X connect 14 0 12 0;
#X connect 15 0 13 0;
#X connect 16 0 10 0;
#X connect 16 1 10 1;
#X connect 17 0 16 0;
#X connect 18 0 17 0;
#X connect 19 0 18 0;
#X restore 57 96 pd crackles;
#N canvas 0 22 450 300 hissing 0;
#X obj 106 188 *~;
#X obj 141 57 lop~ 1;
#X obj 142 107 *~;
#X obj 66 58 hip~ 1000;
#X obj 141 133 *~;
#X obj 142 80 *~ 10;
#X obj 141 157 *~ 600;
#X obj 105 222 outlet~;
#X obj 111 27 inlet~;
#X connect 0 0 7 0;
#X connect 1 0 5 0;
#X connect 2 0 4 0;
#X connect 2 0 4 1;
#X connect 3 0 0 0;
#X connect 4 0 6 0;
#X connect 5 0 2 0;
#X connect 5 0 2 1;
#X connect 6 0 0 1;
#X connect 8 0 1 0;
#X connect 8 0 3 0;
#X restore 149 96 pd hissing;
#N canvas 0 22 170 271 lapping 0;
#X obj 21 109 clip~ -0.9 0.9;
#X obj 21 129 hip~ 25;
#X obj 21 86 hip~ 25;
#X obj 21 40 bp~ 30 5;
#X obj 21 65 *~ 100;
#X obj 21 153 *~ 0.6;
#X obj 20 177 outlet~;
#X obj 21 12 inlet~;
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 2 0 0 0;
#X connect 3 0 4 0;
#X connect 4 0 2 0;
#X connect 5 0 6 0;
#X connect 7 0 3 0;
#X restore 246 97 pd lapping;
#X obj 148 61 noise~;
#X text 203 60 factor common noise;
#X text 305 129 mix levels;
#X obj 151 154 outlet~;
#X obj 57 123 *~ 0.2;
#X obj 246 123 *~ 0.6;
#X obj 150 122 *~ 0.3;
#X text 144 30 fire generator;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 2 0 8 0;
#X connect 3 0 0 0;
#X connect 3 0 1 0;
#X connect 3 0 2 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 6 0;
#X restore 124 46 pd firegen;
#N canvas 769 179 475 599 firegen 0;
#N canvas 8 22 441 619 crackles 0;
#X obj 62 515 *~;
#X obj 126 314 random 30;
#X obj 126 358 line~;
#X obj 126 386 *~;
#X obj 94 446 + 1500;
#X obj 62 470 bp~ 4000 1;
#X obj 93 424 * 500;
#X obj 62 540 outlet~;
#X obj 63 52 inlet~;
#X msg 126 337 1 \, 0 15;
#X obj 126 270 delay 1000;
#X obj 126 290 t b b;
#X obj 126 170 log;
#X obj 126 130 random 2000;
#X obj 126 150 / 2000;
#X obj 126 100 loadbang;
#X obj 126 250 t b f;
#X obj 126 230 min 3000;
#X obj 126 210 max 40;
#X obj 126 190 * -400;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 1 0 6 0;
#X connect 2 0 3 0;
#X connect 2 0 3 1;
#X connect 3 0 0 1;
#X connect 4 0 5 1;
#X connect 5 0 0 0;
#X connect 6 0 4 0;
#X connect 8 0 5 0;
#X connect 9 0 2 0;
#X connect 10 0 11 0;
#X connect 11 0 1 0;
#X connect 11 1 13 0;
#X connect 12 0 19 0;
#X connect 13 0 14 0;
#X connect 14 0 12 0;
#X connect 15 0 13 0;
#X connect 16 0 10 0;
#X connect 16 1 10 1;
#X connect 17 0 16 0;
#X connect 18 0 17 0;
#X connect 19 0 18 0;
#X restore 57 96 pd crackles;
#N canvas 0 22 450 300 hissing 0;
#X obj 106 188 *~;
#X obj 141 57 lop~ 1;
#X obj 142 107 *~;
#X obj 66 58 hip~ 1000;
#X obj 141 133 *~;
#X obj 142 80 *~ 10;
#X obj 141 157 *~ 600;
#X obj 105 222 outlet~;
#X obj 111 27 inlet~;
#X connect 0 0 7 0;
#X connect 1 0 5 0;
#X connect 2 0 4 0;
#X connect 2 0 4 1;
#X connect 3 0 0 0;
#X connect 4 0 6 0;
#X connect 5 0 2 0;
#X connect 5 0 2 1;
#X connect 6 0 0 1;
#X connect 8 0 1 0;
#X connect 8 0 3 0;
#X restore 149 96 pd hissing;
#N canvas 0 22 170 271 lapping 0;
#X obj 21 109 clip~ -0.9 0.9;
#X obj 21 129 hip~ 25;
#X obj 21 86 hip~ 25;
#X obj 21 40 bp~ 30 5;
#X obj 21 65 *~ 100;
#X obj 21 153 *~ 0.6;
#X obj 20 177 outlet~;
#X obj 21 12 inlet~;
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 2 0 0 0;
#X connect 3 0 4 0;
#X connect 4 0 2 0;
#X connect 5 0 6 0;
#X connect 7 0 3 0;
#X restore 246 97 pd lapping;
#X obj 148 61 noise~;
#X text 203 60 factor common noise;
#X text 305 129 mix levels;
#X obj 151 154 outlet~;
#X obj 57 123 *~ 0.2;
#X obj 246 123 *~ 0.6;
#X obj 150 122 *~ 0.3;
#X text 144 30 fire generator;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 2 0 8 0;
#X connect 3 0 0 0;
#X connect 3 0 1 0;
#X connect 3 0 2 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 6 0;
#X restore 217 46 pd firegen;
#N canvas 769 179 475 599 firegen 0;
#N canvas 8 22 441 619 crackles 0;
#X obj 62 515 *~;
#X obj 126 314 random 30;
#X obj 126 358 line~;
#X obj 126 386 *~;
#X obj 94 446 + 1500;
#X obj 62 470 bp~ 4000 1;
#X obj 93 424 * 500;
#X obj 62 540 outlet~;
#X obj 63 52 inlet~;
#X msg 126 337 1 \, 0 15;
#X obj 126 270 delay 1000;
#X obj 126 290 t b b;
#X obj 126 170 log;
#X obj 126 130 random 2000;
#X obj 126 150 / 2000;
#X obj 126 100 loadbang;
#X obj 126 250 t b f;
#X obj 126 230 min 3000;
#X obj 126 210 max 40;
#X obj 126 190 * -400;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 1 0 6 0;
#X connect 2 0 3 0;
#X connect 2 0 3 1;
#X connect 3 0 0 1;
#X connect 4 0 5 1;
#X connect 5 0 0 0;
#X connect 6 0 4 0;
#X connect 8 0 5 0;
#X connect 9 0 2 0;
#X connect 10 0 11 0;
#X connect 11 0 1 0;
#X connect 11 1 13 0;
#X connect 12 0 19 0;
#X connect 13 0 14 0;
#X connect 14 0 12 0;
#X connect 15 0 13 0;
#X connect 16 0 10 0;
#X connect 16 1 10 1;
#X connect 17 0 16 0;
#X connect 18 0 17 0;
#X connect 19 0 18 0;
#X restore 57 96 pd crackles;
#N canvas 0 22 450 300 hissing 0;
#X obj 106 188 *~;
#X obj 141 57 lop~ 1;
#X obj 142 107 *~;
#X obj 66 58 hip~ 1000;
#X obj 141 133 *~;
#X obj 142 80 *~ 10;
#X obj 141 157 *~ 600;
#X obj 105 222 outlet~;
#X obj 111 27 inlet~;
#X connect 0 0 7 0;
#X connect 1 0 5 0;
#X connect 2 0 4 0;
#X connect 2 0 4 1;
#X connect 3 0 0 0;
#X connect 4 0 6 0;
#X connect 5 0 2 0;
#X connect 5 0 2 1;
#X connect 6 0 0 1;
#X connect 8 0 1 0;
#X connect 8 0 3 0;
#X restore 149 96 pd hissing;
#N canvas 0 22 170 271 lapping 0;
#X obj 21 109 clip~ -0.9 0.9;
#X obj 21 129 hip~ 25;
#X obj 21 86 hip~ 25;
#X obj 21 40 bp~ 30 5;
#X obj 21 65 *~ 100;
#X obj 21 153 *~ 0.6;
#X obj 20 177 outlet~;
#X obj 21 12 inlet~;
#X connect 0 0 1 0;
#X connect 1 0 5 0;
#X connect 2 0 0 0;
#X connect 3 0 4 0;
#X connect 4 0 2 0;
#X connect 5 0 6 0;
#X connect 7 0 3 0;
#X restore 246 97 pd lapping;
#X obj 148 61 noise~;
#X text 203 60 factor common noise;
#X text 305 129 mix levels;
#X obj 151 154 outlet~;
#X obj 57 123 *~ 0.2;
#X obj 246 123 *~ 0.6;
#X obj 150 122 *~ 0.3;
#X text 144 30 fire generator;
#X connect 0 0 7 0;
#X connect 1 0 9 0;
#X connect 2 0 8 0;
#X connect 3 0 0 0;
#X connect 3 0 1 0;
#X connect 3 0 2 0;
#X connect 7 0 6 0;
#X connect 8 0 6 0;
#X connect 9 0 6 0;
#X restore 317 46 pd firegen;
#X connect 0 0 5 0;
#X connect 1 0 5 0;
#X connect 2 0 5 0;
#X connect 3 0 5 0;
#X connect 4 0 3 0;
#X connect 5 0 6 1;
#X connect 5 0 6 0;
#X connect 7 0 1 0;
#X connect 8 0 2 0;
#X connect 9 0 0 0;
