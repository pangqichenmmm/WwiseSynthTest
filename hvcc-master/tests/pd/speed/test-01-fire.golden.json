{
  "samplerate": 48000.0,
  "blocksize": 512,
  "numIterations": 2000000,
  "usPerBlock": {
    "HV_SIMD_SSE": 27.123465
  }
}
